kafka_utils/kafka_consumer_manager/commands/offset_get.py

```diff
@@ -148,3 +160,26 @@ def print_output(cls, consumer_offsets_metadata, watermark_filter):
                             current=metadata_tuple.current
                         )
                     )
+                if watermark_filter == "all" or watermark_filter == "distance":
+                    per_distance = cls.percentage_distance(metadata_tuple.highmark, metadata_tuple.current)
```

Comment:
Why do we need to cast to int? Can't we cast to int once here when we invoke the function instead than performing the cast multiple times inside percentage_distance()?